etc/org.opencastproject.speechtotext.impl.SpeechToTextServiceImpl.cfg

```diff
@@ -4,4 +4,4 @@
 
 # Select STT engine.
 # Available engines: vosk, whisper, whispercpp | Default: vosk
-#SpeechToTextEngine.target=(enginetype=vosk)
+SpeechToTextEngine.target=(enginetype=whisper)
```

Comment:
A few of issues here:
- You meant `whispercpp` here I'm assuming?
- This changes the default config, and thus can't go into `r/15.x`
- This config change still does not work how you want it to.

`SpeechToTextServiceImpl` does not read this config and thus can't actually change its binding filter.  You want to add logic to its `activate` method, where both the `@activate` and `@modified` annotations live.  And probably rip out/replace `setSpeechToTextEngine` in the same class, since it's where the hardcoded binding lives.

Reply:
This PR does NOT intend to change the default SpeechToText-Engine - the changed line just reflects the changes from 14 to 15.

Reply:
I think it's probably https://github.com/opencast/opencast/pull/4973/commits/014827467f70d8a650cdefc6063dc2327fea8d6f that's causing the problem.  I saw that listed in this PR, and assumed it was an attempt to resolve the binding issue from the commit log entry.  Your commit seems to duplicate 0e2c9ba3086a2d6093a5ed3f28709cfb149de1e3 (which probably shouldn't have touche this file, but I digress), which is already in r/15.x